README.md

```diff
@@ -16,7 +16,7 @@ React has been designed for gradual adoption from the start, and **you can use a
 * [Add React to a Website](https://reactjs.org/docs/add-react-to-a-website.html) as a `<script>` tag in one minute.
 * [Create a New React App](https://reactjs.org/docs/create-a-new-react-app.html) if you're looking for a powerful JavaScript toolchain.
 
-You can use React as a `<script>` tag from a [CDN](https://reactjs.org/docs/cdn-links.html), or as a `react` package on [npm](https://www.npmjs.com/package/react).
+You can use React as a `<script>` tag from a [CDN](https://reactjs.org/docs/cdn-links.html) or as a `react` package on [npm](https://www.npmjs.com/package/react).
```

Comment:
Not sure if we need to remove the comma here, two independent sentence can have comma before the `or`